data/json/snippets.json

```diff
@@ -1791,6 +1791,39 @@
     "category": "note",
     "text": [ "\"Husband 30 yrs died in bombs.  Mushrooms brought him back.  GIVE YOURSELF TO THEm AND THEY WILL HELP YOU\"" ]
   },
+  {
```

Comment:
Cut all these snippets, none of them make sense to appear in the dda game setting.